addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -647,6 +647,12 @@ def create(self, vals_list):
         allocations = super(HolidaysAllocation, self.with_context(mail_create_nosubscribe=True)).create(vals_list)
         allocations._add_lastcalls()
         for allocation in allocations:
+            # We recompute the description to have a value for the private_name field
+            # because the inverse method is not triggered on creation even if the field is
+            # in the view
+            if self._context.get('form_view_ref') == 'hr_holidays.hr_leave_allocation_view_form_dashboard':
+                allocation._compute_description()
+                allocation.sudo().private_name = allocation.name
```

Comment:
Seems like there is a missing trigger on the compute method. 

We shouldn't call manually compute methods.

Reply:
Yup true I just add the form_view_ref as a trigger and everything is working fine